Message/show.html

```diff
@@ -272,9 +275,12 @@
       <div class="col-md-3">
        <a data-privly-chrome="https://chrome.google.com/webstore/detail/pkokikcdapfpkkkjpdaamjanniaempol" data-privly-firefox="https://addons.mozilla.org/en-US/firefox/addon/privly/" href="https://">
         <div style="position:relative;width:10em;padding:1em;height:150px">
-         <img alt="Google Chrome Logo" class="download-extension-img" id="chrome_img" src="../shared/images/chrome-logo_150.png"/>
-         <img alt="Firefox Logo" class="download-extension-img" id="firefox_img" src="../shared/images/firefox-logo_150.png"/>
-         <img alt="Privly Icon" src="../shared/images/icon_96.png" style="position:absolute;right:33px;top:15px;"/>
+         <img alt="Google Chrome Logo" class="download-extension-img" id="chrome_img" src="../shared/images/chrome-logo_150.png">
```

Comment:
Yeah, the rest seems good: un-closed `<meta>` tags are closed automatically, etc.